### ISSUE TITLE
terrain

### ISSUE THREAD

**Issue by UM004** [leombastos/2023_AdvPA#67]
Hi everyone,
I am having problem with elevation maps. Can anyone kindly help? My elevation map is going beyond the boundary
![Screenshot (328)](https://user-images.githubusercontent.com/122295650/235986819-45241614-aa94-4f0d-9428-84c245d81901.png)
![Screenshot (329)](https://user-images.githubusercontent.com/122295650/235986852-6a43ce88-26be-4a5b-8eb1-c4a367825233.png)

**Comment by Trippsmith55** [comment]
Try adding    st_crop(boundary_w)     after     st_rasterize(dx = 10, dy = 10)%>%

**Comment by thapashekhar** [comment]
@Trippsmith55, you are right. Not having st_crop(boundary_w) on the code, grid_r will be a rectangular grid, which will be way bigger than the field boundary and once you interpolated using this grid_r, it will predict for all grid cells which are also outside the field boundary.

**Comment by UM004** [comment]
Thank you @thapashekhar and @Trippsmith55 .
